### ISSUE TITLE
Window decorations are not rendered correctly

### ISSUE THREAD

**Issue by alexkreidler** [papyros/papyros-shell#210]
This is what they look like. They should look different. (I checked an SVG preview of the raw files)

![image](https://cloud.githubusercontent.com/assets/11166947/12071976/3643a74c-b096-11e5-84ab-23a0c783693d.png)
##

**Comment by iBelieve** [comment]
Was this a cropped version of the screenshot from the issue about the decorations not scaling? I'm guessing the issue is the images are two small at that decoration size to draw properly.

**Comment by alexkreidler** [comment]
No, it was a different one that I took after testing, but I did have it scaled up at the time. I will test with the settings scaled down to see if it keeps happening.

**Comment by alexkreidler** [comment]
Just re-tested with the display scaled at 100%, and it still happens.

**Comment by alexkreidler** [comment]
Recent screenshot:
![papyros-scaling](https://cloud.githubusercontent.com/assets/11166947/12073106/e4834eb6-b0d7-11e5-9080-eaedd941d0f4.png)

**Comment by Tim15** [comment]
I have another issue with window decorations:
When I build from source and run `./papyros-session` from the build directory, the window decorations look fine.
When I run`sudo make install` and run `papyros-session` (therefore `/usr/bin/papyros-session`), it uses the old ugly window decorations.
